lib/value.ts

```diff
@@ -57,6 +57,41 @@ function napi_create_object (env: napi_env, result: Pointer<napi_value>): emnapi
   })
 }
 
+function napi_create_array (env: napi_env, result: Pointer<napi_value>): emnapi.napi_status {
+  return emnapi.checkEnv(env, (envObject) => {
+    return emnapi.checkArgs(env, [result], () => {
+      HEAP32[result >> 2] = envObject.getCurrentScope().add([]).id
+      return emnapi.napi_clear_last_error(env)
+    })
+  })
+}
+
+function napi_create_array_with_length (env: napi_env, length: size_t, result: Pointer<napi_value>): emnapi.napi_status {
+  return emnapi.checkEnv(env, (envObject) => {
+    return emnapi.checkArgs(env, [result], () => {
+      HEAP32[result >> 2] = envObject.getCurrentScope().add(new Array(length)).id
+      return emnapi.napi_clear_last_error(env)
+    })
+  })
+}
+
+function napi_create_symbol (env: napi_env, description: napi_value, result: Pointer<napi_value>): emnapi.napi_status {
+  return emnapi.checkEnv(env, (envObject) => {
+    return emnapi.checkArgs(env, [result], () => {
+      if (description === emnapi.NULL) {
+        HEAP32[result >> 2] = envObject.getCurrentScope().add('').id
+      } else {
+        const desc = UTF8ToString(description)
```

Comment:
`description` is not a `char*` so no need to use `UTF8ToString()` here. Just get the JS value from the `handleStore` in `envObject`